external/fv3fit/fv3fit/emulation/thermobasis/emulator.py

```diff
@@ -82,6 +82,7 @@ class Config:
     output_path: str = ""
     weight_sharing: bool = False
     cloud_water: bool = False
+    l2: Optional[float] = None
```

Comment:
nit: If you set this as type `float` with default value 0, it could simplify the code somewhat, though this is less of a concern since Keras accepts None wherever regularizers are used.